lgc/patch/PatchEntryPointMutate.cpp

```diff
@@ -666,7 +666,7 @@ void PatchEntryPointMutate::processShader(ShaderInputs *shaderInputs) {
       std::min(resUsage->numSgprsAvailable, m_pipelineState->getTargetInfo().getGpuProperty().maxSgprsAvailable);
 
   if (shaderOptions->maxThreadGroupsPerComputeUnit != 0) {
-    std::string wavesPerEu = std::string("0,") + std::to_string(shaderOptions->maxThreadGroupsPerComputeUnit);
+    std::string wavesPerEu = std::string("1,") + std::to_string(shaderOptions->maxThreadGroupsPerComputeUnit);
```

Comment:
nit: llvm::Twine would work well here:
```
std::string wavesPerEu = ("1," + Twine(shaderOptions->maxThreadGroupsPerComputeUnit)).str();
```

Reply:
I'm sure twinifying things is great but could you do it as a separate series of NFC patches? I think it would obscure simple patches like this to do it at the same time.

Reply:
Sure, I always use nits as optional changes